### PR TITLE
Add namespaceSelector:matchLabels example

### LINKED ISSUE #40858: Issue with k8s.io/docs/concepts/services-networking/network-policies/
The page is not covering a pretty common case of targeting multiple namespaces by name. I need an egress policy allowing to send to pods in two namespaces, cannot find any way of doing it, will have to search on Google and hit tons of StackOverflow posts suggesting random solutions that may or may not work. I don't have enough privileges on the Kubernetes cluster to add labels to namespaces in order to search by label. This could be a lack of flexibility of network policies, or something missing from the documentation.

### PR DESCRIPTION
**Pull request by mrgiles**
Adds a new section with an example showing the use of  multiple namespaceSelector for an Egress policy. Fixes #40858

### CODE REVIEW

File: content/en/docs/concepts/services-networking/network-policies.md [hunk 1]
```diff
@@ -300,6 +300,45 @@ does not support the `endPort` field and you specify a NetworkPolicy with that,
...
+spec:
+  podSelector:
+    matchLabels:
+      app: myapp
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - namespaceSelector:
+        matchLabels:
```
Comment (tengqm) [inline content/en/docs/concepts/services-networking/network-policies.md:329]: I'd suggest we use `matchExpressions` here for clarity. E.g.

```yaml
matchExpressions:
  - key: namespace
    operator: In
    values: ["frontend", "backend"]
```

The effect is the same but the readability is a little bit different.
You can provide multiple `NetworkPolicyIngressRule` in the `to` list, however, readers must be aware that these items are logical OR'ed during evaluation.

Comment (mrgiles) [reply content/en/docs/concepts/services-networking/network-policies.md]: Done. Thanks for the suggestion.

**Comment by mrgiles** [comment]
Thank you for your review @tengqm. Can you add the **lgtm** label so that the PR can be merged? Thanks!

**Comment by tengqm** [comment]
> Thank you for your review @tengqm. Can you add the **lgtm** label so that the PR can be merged? Thanks!

I'd avoid 'lgtm' AND 'approve' a PR at the same time, unless a PR has been dangling too long to be approved, or I'm 100% confident from both docs and tech's perspective. It is always good to have another pair of eyes to screen a PR, by the way.

**Comment by mrgiles** [comment]
Good point @tengqm. I thought that **approved** always came after **lgtm**, I didn't think they could be swapped. Thanks for the clarification.

**Comment by Shubham82** [comment]
Thanks @mrgiles for the PR.
It looks good to me.
/lgtm

**Pull request merged**